### PR TITLE
Get rid of broken activate scripts

### PR DESCRIPTION
**Pull request by mingwandroid**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Fixes https://github.com/conda-forge/python-libarchive-c-feedstock/issues/6

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+  sha256: {{ sha256 }}
   patches:
     - 0001-Add-zstd-support.patch
 
 build:
-  number: 1002
+  number: 1003
 
 requirements:
   host:
```
Comment (mariusvniekerk) [inline recipe/meta.yaml:24]: can we add some runtime requirements that ensure we have binutils or a lightweight version of it around?

**Comment by bollwyvl** [comment]
Thanks! :crossed_fingers:

**Comment by mariusvniekerk** [comment]
This is probably the solution we want https://github.com/conda-forge/python-feedstock/pull/205

**Comment by bollwyvl** [comment]
@mariusvniekerk @mingwandroid i defer to you folks on how we wish to proceed!

**Comment by mingwandroid** [comment]
The only reason we need binutils is to get the soname. I already have code that does that (pyldd/overthinking).. PR later for review. I'm no Python coder though but we should attempt to upstream this functionality into ctypes after some cleaning!

File: recipe/bld.bat [hunk 2]
```diff
@@ -1,14 +1,2 @@
...
-set DEACTIVATE_DIR=%PREFIX%\etc\conda\deactivate.d
-mkdir %ACTIVATE_DIR%
-mkdir %DEACTIVATE_DIR%
-
-copy %RECIPE_DIR%\scripts\activate.bat %ACTIVATE_DIR%\python-libarchive-c-activate.bat
-if errorlevel 1 exit 1
-
-copy %RECIPE_DIR%\scripts\deactivate.bat %DEACTIVATE_DIR%\python-libarchive-c-deactivate.bat
-if errorlevel 1 exit 1
+python setup.py install --single-version-externally-managed --record record.txt
```
Comment (ocefpaf) [inline recipe/bld.bat]: I'm confused. The issue was pip or the activation?
I could not test it on OS X locally but Linux and Windows were passing and Travis-CI was OK :-/

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:24]: :+1:

File: recipe/bld.bat (hunk 2 above)
Comment (ocefpaf) [reply recipe/bld.bat]: Answering my own questions, it looks like we don't have `${SHLIB_EXT}` defined outside of the build scope :-(

xref: https://github.com/conda-forge/python-libarchive-c-feedstock/issues/6#issuecomment-428023824

Comment (mingwandroid) [reply recipe/bld.bat]: Oh I find PIP a bit of a pain, most recently not creating entry point scripts on Windows. Feel free to revert this part if pip is the official policy (though I'd propose a policy where we use the simplest tool that's capable of building a correct package in each case).

Comment (mingwandroid) [reply recipe/bld.bat]: The activate script serves no useful purpose here, we need to just kill them.

**Comment by marscher** [comment]
win, py27 build did not succeed.

**Comment by jakirkham** [comment]
Triggered a rebuild for it. If that doesn't work, could you please file an issue, @marscher?

**Comment by jakirkham** [comment]
NVM the rebuild failed quickly. Have raised in issue ( https://github.com/conda-forge/python-libarchive-c-feedstock/issues/8 ).

**Pull request merged**